src/docgen/view/parameter.ts

```diff
@@ -24,6 +24,7 @@ export class Parameter {
       optional: this.transpiledParam.optional === true ? true : undefined, // to save space
       default: this.parameter.spec.docs?.default,
       type: this.transpiledParam.typeReference.toJson(),
+      variadic: this.parameter.spec.variadic ?? false,
```

Comment:
```suggestion
      variadic: this.transpiledParam.variadic,
```

Reply:
At a high level, the way docgen works is:

`jsii.spec -> jsii.reflect -> transpiled -> schema -> markdown`

As we discussed, the markdown rendering should be language agnostic, where the schema contains every language specific formatting. Is adding `variadic` to the schema enough for the renderer to know how to format types? its ok to stick it in here as possible useful metadata, but its not enough (this is why metadata is still incorrect).